test/common/tests.go

```diff
@@ -28,6 +28,7 @@ var (
 		{"E05 - Verify Grafana Route returns dashboards", TestGrafanaExternalRouteDashboardExist},
 		{"C04 - Verify Alerts exist", TestIntegreatlyAlertsExist},
 		{"C01 - Verify Alerts are not firing apart from DeadMansSwitch", TestIntegreatlyAlertsFiring},
+		{"C01.1 - Verify Alerts are not firing apart from DeadMansSwitch", TestIntegreatlyAlertsPendingOrFiring},
```

Comment:
Could we change this to `Verify Alerts are not firing during or after intsallation apart from DeadMansSwitch`

Reply:
yeah i didnt know what to change it to